### PR TITLE
Styles initial UI for chat feature

### PR DESCRIPTION
**Pull request by vlop11**
This branch works on styling the UI of the chat feature. This may change over time, but this a basic idea of how it should look.

(Note: the next goal is to have a top nav bar with links to map, calendar, and groups page)
chat

### CODE REVIEW

File: src/main/java/templates/chat.soy [hunk 1]
```diff
@@ -7,20 +7,57 @@
...
+                        </div>
+      
+                    </nav>
+                </div>
+            </aside>
+
+            <main class="main-content">
+                <ul class="mdc-list messages-container">
+                    {for $message in $messages}
+                    <li class="mdc-list-item" tabindex="0">
```
Comment (ahlschlager) [inline src/main/java/templates/chat.soy]: indent inside the for loop

File: src/main/webapp/chat.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
+body {
+    display: flex;
+    flex-direction: row;
+    margin: 0px;
```
Comment (dylancastillo) [inline src/main/webapp/chat.css]: nit: remove px after 0 throughout the file

File: src/main/webapp/chat.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    margin: 10px 0px 0px 0px;
+    padding: 0 16px;
+}
+
+.group-label {
+    font-size: 12pt;
+}
+
+.main-content {
+    margin: auto;
```
Comment (dylancastillo) [inline src/main/webapp/chat.css]: nit: alphabetize

File: src/main/webapp/updateChat.js [hunk 4]
```diff
@@ -7,10 +10,11 @@ function init() {
 
     fetch(urlString).then(response => response.json()).then(json => {
         for (let i = 0; i < json.length; i++) {
-            const p = document.createElement("P");
-            p.innerText = json[i];
-            p.classList.add("message");
-            messagesContainer.appendChild(p);
+            const messageTemplate = templates.message.messageBubble({
+                "message": json[i]
+            });
```
Comment (courtneywong) [inline src/main/webapp/updateChat.js]: I think the correct formatting here is 
```
const messageTemplate =
          templates.message.messageBubble({'message': json[i]});
```

**Review by courtneywong — CHANGES_REQUESTED** [review]

**Review by courtneywong — APPROVED** [review]

**Review by sporkert — APPROVED** [review]

**Pull request merged**